api/orchestrator/orchestrator.proto

```diff
@@ -749,7 +745,8 @@ message Category {
   repeated Control controls = 4 [
```

Comment:
I think we need to remove the 

  repeated Catalog catalogs_in_scope = 4 [
    (tagger.tags) = "gorm:\"many2many:audit_scopes\"",
    (buf.validate.field).repeated.items.required = true,
    (google.api.field_behavior) = REQUIRED
  ];

in line 644 (I cannot place my comment there unfortunately).

Reply:
I've deleted the catalogs_in_scope. I hope I've updated everything necessary.